### PR TITLE
Layout & theme update (WIP)

### PR DESCRIPTION
**Pull request by jongbonga**
Update the landing layout to include the organisation & ripple's logo.
Update theme & remove nautilus colors

### CODE REVIEW

**Review by billguo99 — COMMENTED** [review]
image
Palau theming is good.

image
Nautilus theming just needs a fix in the ShowAsset part :)

Otherwise very cool, love the new theming colors.

**Comment by billguo99** [comment]
I ran `npm test` and a couple of tests failed.
```
SUMMARY:
✔ 385 tests completed
ℹ 4 tests skipped
✖ 5 tests failed

FAILED TESTS:
  WalletAccessPage
    onPinConfirmed
      ✖ opens wallet
        Chrome 106.0.0.0 (Linux x86_64)
      Error: Timeout - Async function did not complete within 5000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
          at <Jasmine>

      ✖ shows open wallet error
        Chrome 106.0.0.0 (Linux x86_64)
      Error: Timeout - Async function did not complete within 5000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
          at <Jasmine>

  KycPage
    ✖ onSubmit
      Chrome 106.0.0.0 (Linux x86_64)
    Error: Timeout - Async function did not complete within 5000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
        at <Jasmine>

    ✖ onSdkComplete + createCheck
      Chrome 106.0.0.0 (Linux x86_64)
    Error: Timeout - Async function did not complete within 5000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
        at <Jasmine>

    ✖ refreshCheck
      Chrome 106.0.0.0 (Linux x86_64)
    Error: Timeout - Async function did not complete within 5000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
        at <Jasmine>
```

File: web-client/src/app/app.component.ts [hunk 1]
```diff
@@ -1,10 +1,39 @@
...
-  constructor() {}
+  constructor(private platform: Platform) {
+    this.initializeApp();
+  }
+
+  initializeApp() {
+    this.platform
+      .ready()
+      .then(async () => {
+        this.loadTheme(environment.organization);
```
[the remark this answers is not in the document]
Comment (jongbonga) [reply web-client/src/app/app.component.ts:20]: Yes, I need to add the splash screen for the natives. The splash needs to wait until the theme is applied before it disappears.

let me add the splash as part of this PR

**Comment by jongbonga** [comment]
> I ran `npm test` and a couple of tests failed.
> 
> ```
> SUMMARY:
> ✔ 385 tests completed
> ℹ 4 tests skipped
> ✖ 5 tests failed
> 
> FAILED TESTS:
>   WalletAccessPage
>     onPinConfirmed
>       ✖ opens wallet
>         Chrome 106.0.0.0 (Linux x86_64)
>       Error: Timeout - Async function did not complete within 5000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
>           at <Jasmine>
> 
>       ✖ shows open wallet error
>         Chrome 106.0.0.0 (Linux x86_64)
>       Error: Timeout - Async function did not complete within 5000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
>           at <Jasmine>
> 
>   KycPage
>     ✖ onSubmit
>       Chrome 106.0.0.0 (Linux x86_64)
>     Error: Timeout - Async function did not complete within 5000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
>         at <Jasmine>
> 
>     ✖ onSdkComplete + createCheck
>       Chrome 106.0.0.0 (Linux x86_64)
>     Error: Timeout - Async function did not complete within 5000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
>         at <Jasmine>
> 
>     ✖ refreshCheck
>       Chrome 106.0.0.0 (Linux x86_64)
>     Error: Timeout - Async function did not complete within 5000ms (set by jasmine.DEFAULT_TIMEOUT_INTERVAL)
>         at <Jasmine>
> ```
> 
> Did these tests succeed for you?

They didn't fail on Github

**Comment by jongbonga** [comment]
> image Palau theming is good.
> 
> image Nautilus theming just needs a fix in the ShowAsset part :)
> 
> Otherwise very cool, love the new theming colors.

Will look into the asset item now. nice catch, thank you

**Review by billguo99 — APPROVED** [review]

**Review by billguo99 — APPROVED** [review]

**Pull request merged**